docs/massivekb.md

```diff
@@ -4,7 +4,30 @@ The MassIVE Knowledge Base (MassIVE-KB) is a set of peptide spectral libraries.
 
```

Comment:
The word "peek" is misspelled as "peak" in the introductory paragraph above. For some reason I could not tag the actual line containing that paragraph with this comment.